assembl/alembic/versions/41176a6a6758_tombstonable_content.py

```diff
@@ -0,0 +1,74 @@
+"""tombstonable Content
+
+Revision ID: 41176a6a6758
+Revises: 5eb326c815de
+Create Date: 2016-07-26 18:00:28.194728
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '41176a6a6758'
+down_revision = '5eb326c815de'
+
+from alembic import context, op
+import sqlalchemy as sa
+import transaction
+
+
+def upgrade(pyramid_env):
+    with context.begin_transaction():
+        op.add_column('content', sa.Column('tombstone_date', sa.DateTime))
+
+
+    from assembl import models as m
```

Comment:
Wouldn't it be better to declare imports at the beginning of file for readability purposes?